softdev.bib

```diff
@@ -1919,6 +2008,26 @@ @inproceedings{vanter00displaying
     month = Jun,
 }
 
+@inproceedings{vanter94practical,
+  title = {Practical language-based editing for software engineers},
+  author = {Van De Vanter, Michael L},
+  booktitle = {{W}orkshop on Software Engineering and Human-Computer Interaction},
```

Comment:
Does that workshop have an abbreviation? If so let's use it. Otherwise, let's drop the curly brackets around "W".

Reply:
Ah, we're not supposed to put "workshop" in the title. It should just be "Software Engineering and Human-Computer Interaction" (or, preferably, if that workshop has an abbreviation, just use that).

Reply:
I see. I fixed it by using the abbreviation.